agent/instrumentation/applicationinsights-web-2.3/build.gradle.kts

```diff
@@ -12,12 +12,12 @@ muzzle {
 }
 
 val otelInstrumentationAlphaVersion: String by project
-val otelVersion: String by project
+val otelInstrumentationVersion: String by project
 
 dependencies {
   compileOnly("com.microsoft.azure:applicationinsights-web:2.3.0")
 
-  testImplementation("io.opentelemetry.instrumentation:opentelemetry-instrumentation-annotations:$otelVersion")
+  testImplementation("io.opentelemetry.instrumentation:opentelemetry-instrumentation-annotations:$otelInstrumentationVersion")
```

Comment:
can you put a note saying when it's moved from sdk to otelInstrumentation?

Reply:
> can you put a note saying when it's moved from sdk to otelInstrumentation?

I didn't follow this comment, can you clarify? thanks

Reply:
this has been there for 2 years. I just want to know when it was moved to a different artifact and put a comment there.

Reply:
ah, it has always been wrong

it just happened to work because before we bumped instrumentation repo to 2.x it generally had the same version number as the SDK repo

Reply:
plus, it's still "working" (CI passes in this repo), it just spits out a warning which I think S360 may be complaining about